skbio/__init__.py

```diff
@@ -79,3 +77,8 @@
     __doc__ = title + art
 else:
     __doc__ = title + art + __doc__
+
+test = TestRunner(__file__).test
+
+if __name__ == '__main__':
+    test()
```

Comment:
Turns out nose conveniently has a ``run_exit`` method which does what you would expect.